docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  backend:
+    build: .
+    ports:
+      - "8000:8000"
+    expose:
+      - "8000"
+  frontend:
+    build: ../frontend/deaddrop/.
```

Comment:
for now add a comment mentioning something along the lines of this assuming a specific directory structure, where the frontend is in the same parent folder as the backend

i think this may actually be fine in the long-run, even if technically janky. usually this wld be solved by having everything in the same repo, which we can do with git submodules